.gitignore

```diff
@@ -11,3 +11,4 @@ deps/pdf2svg.svg
 deps/showed_warning
 /Manifest.toml
 /test/coverage/Manifest.toml
+.DS_Store
```

Comment:
I would recommend that you just delete or ignore `.DS_Store` instead of adding it to the ignore list of this repo. AFAIK it is an OS X temporary directory. You can also ignore it for all git repos on your computer, search for "Patterns which a user wants Git to ignore in all situations" in `man gitignore`

Reply:
Didn't know about that, cool! I'll fix it.